app/views/vocab_sheets/_vocab_sheet_item.html.haml

```diff
@@ -5,9 +5,9 @@
       = link_to sign_url(vocab_sheet_item.sign_id), { class: "drawing vocab_sheet_drawing drawing vocab_sheet_drawing__size--#{size}" } do
         = image_tag "signs/#{vocab_sheet_image_width}-#{vocab_sheet_image_height}/#{vocab_sheet_item.drawing}"
       = form_tag vocab_sheet_item_path(vocab_sheet_item), method: :put do
-        = text_area_tag "item[name]", vocab_sheet_item.name, class: 'item-name vocab-sheet__text-input h3', id: "item_#{vocab_sheet_item.id}_name"
+        = text_area_tag "item[name]", vocab_sheet_item.name, class: 'item-name vocab-sheet__text-input h3', id: "item_#{vocab_sheet_item.id}_name", maxLength: "100"
         = hidden_field_tag "item[#{vocab_sheet_item.id}][old_name]", vocab_sheet_item.name, class: 'old_name'
-        = text_area_tag "item[maori_name]", vocab_sheet_item.maori_name, class: 'item_maori_name vocab-sheet__text-input h3 maori_gloss', id: "item_#{vocab_sheet_item.id}_maori_name"
+        = text_area_tag "item[maori_name]", vocab_sheet_item.maori_name, class: 'item_maori_name vocab-sheet__text-input h3 maori_gloss', id: "item_#{vocab_sheet_item.id}_maori_name", maxLength: "100"
```

Comment:
Just a small thought: Maybe this is best done as a global variable or something set by the `Item` controller?
🐘

Reply:
what is best done as a global variable?

Reply:
I think he's talking about the value for max length.
I agree but instead of doing it in the controller or as a global it could just be a small wrapper method on the item model itself given that we already have access to it in the form of "vocab_sheet_item" and it should probably be a responsibility of that model anyway.
That way you could call something like vocab_sheet_item.max_length_for_name

Reply:
its a element attribute right?

Reply:
Instead of putting in the controller, can it go in `VocabSheetHelper`.
```
def max_field_length
  100
end
```

Reply:
None of it is enforced on the model though....